sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -131,41 +131,59 @@ var adminClient = new MetricsAdvisorAdministrationClient(new Uri(endpoint), new
 
 ### MetricsAdvisorAdministrationClient
```

Comment:
I can't comment, but the Readme has version 1.0.0 https://github.com/Azure/azure-sdk-for-net/blob/2c7f296311a6d193a1c1da1a6be12caf0e439a71/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md#install-the-package but when we Ga we don't have the version so when the user reads this, they can get the latest package

Reply:
Removed the version from README.